modules/ops/conf/conf.go

```diff
@@ -21,6 +21,7 @@ import (
 // Conf Define the configuration
 type Conf struct {
 	Debug         bool   `env:"DEBUG" default:"false"`
+	EnableEss     bool   `env:"ENABLE_ESS" default:"false"`
```

Comment:
What scenarios need to be opened? Where to configure?

Reply:
It's a ecs auto scaling feature come from exxonmobil, but not used, and can be enabled by dice yaml environment.